lib/core/feature-flags/src/lib/components/flags/flags.component.html

```diff
@@ -0,0 +1,70 @@
+<mat-toolbar class="feature-flags-overrides-header">
+  <div class="feature-flags-overrides-header-text">
+    <adf-feature-flags-override-indicator></adf-feature-flags-override-indicator>
+    <span>{{ "CORE.FEATURE-FLAGS.OVERRIDE" | translate }}</span>
+  </div>
+  <mat-slide-toggle
+    color="warning"
+    [checked]="isEnabled"
+    (change)="onEnable($event.checked)">
+  </mat-slide-toggle>
+  <button
+    class="feature-flags-overrides-header-close"
+    mat-icon-button
+    mat-dialog-close>
+        <mat-icon>close</mat-icon>
+  </button>
+</mat-toolbar>
+
+<ng-container *ngIf="flags$ | async as flags">
+  <table mat-table [dataSource]="flags" class="feature-flags-overrides-table mat-elevation-z0">
+    <ng-container matColumnDef="icon">
+      <th mat-header-cell class="icon-col header-cell" *matHeaderCellDef>
+        <mat-icon class="material-icons-outlined" fontIcon="search" class="search-icon"></mat-icon>
+      </th>
+      <td mat-cell class="icon-col" *matCellDef="let element">
+        <button mat-icon-button *ngIf="element.fictive; else flagFromApi" class="fictive-flag-button" (click)="onDelete(element.flag)">
+          <mat-icon class="material-icons-outlined custom-flag-icon" fontIcon="memory"></mat-icon>
```

Comment:
this button has 2 icons instead of 1, is that intended?

Reply:
yes. Trash icon is displayed on hover event
```
:hover {
            .custom-flag-icon {
                display: none;
            }

            .trash-icon {
                display: block;
            }
        }
```